po4a

```diff
@@ -388,6 +388,10 @@ Do not generate the translated documents, only update the POT and PO files.
 
 Do not change the POT and PO files, only the translation may be updated.
 
+=item B<--no-rm-translations>
+
+Keeps the old translation files if the translation doesn't meet the threshold specified by --keep.
```

Comment:
You should better explain the effect of this option: look how long it took me to understand :)

Plus, you should warn the users that if they activate this option, they may end up with well translated but outdated translation files. That's very different from the usual behavior of po4a and should be stressed in the doc.

Reply:
There's nothing like 100% understanding your own thoughts and being surprised when others don't do the same. Having experienced our discussion I was expecting this comment :-)